.github/workflows/ci.yml

```diff
@@ -142,9 +142,16 @@ jobs:
         id: electron-unit-tests
         run: DISPLAY=:10 ./scripts/test.sh --runGlob "**/sql/**/*.test.js" --coverage
 
-      - name: Run Extension Unit Tests # {{SQL CARBON EDIT}} Rename to core for clarity
-        id: electron-extension-unit-tests
+      - name: Run Extension Unit Tests (Continue on Error) # {{SQL CARBON EDIT}} Rename to core for clarity
+        id: electron-extension-unit-tests-continue-on-error
         run: DISPLAY=:10 ./scripts/test-extensions-unit.sh
+        continueOnError: true
+        condition: and(succeeded(), ne(variables['EXTENSION_UNIT_TESTS_FAIL_ON_ERROR'], 'true'))
```

Comment:
The default for these should be to fail on error, so I'd suggest reversing these checks to eq 'false' for this one and ne 'true' for the one below